README.md

```diff
@@ -4,6 +4,7 @@ Provides a PHP CodeSniffer ruleset for the MO4 coding standard
 
 [![Build Status](https://travis-ci.org/mayflower/mo4-coding-standard.svg?branch=master)](https://travis-ci.org/mayflower/mo4-coding-standard)
 [![Build Status](https://codecov.io/gh/mayflower/mo4-coding-standard/branch/master/graph/badge.svg)](https://codecov.io/gh/mayflower/mo4-coding-standard/branch/master/)
+[![Scrutinizer Quality Level](https://scrutinizer-ci.com/g/mayflower/mo4-coding-standard/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/mayflower/mo4-coding-standard)
```

Comment:
I recommend to deactivated checking of the `MO4/Tests/*/*.inc` files. They are often broken on purpose 😸